ucsb_ds_capstone_projects_2021/projects/ccber/update1.md

```diff
@@ -1,13 +1,22 @@
 # Update 1
-## Week 1
+## Overview
+For the first half of this winter quarter, we focused on doing exploratory analysis on our [Global Biotic Interactions (GloBI)](https://www.globalbioticinteractions.org/about) dataset. This dataset contains information on interactions between bees and plants. Each row represents an interaction, and the data comes from museum records, publications, and from people submitting their sightings/observations to be recorded online. There are columns detailing the taxonomy of the bees and the plants, like what family, genus, or species they belong to. There is also a column detailing what is the source of this sighting, which is the citation. 
+
+Our goal is to understand interactions between bees and flowers, like which types of bees visit which types of flowers and why. To accomplish this, we made network graphs and heatmaps with various R and Python packages like [igraph](https://igraph.org/r/), [bipartite](https://www.rdocumentation.org/packages/bipartite/versions/2.16), [NetworkX](https://networkx.org/), [Plotly](https://plotly.com/python/), [ggraph](https://cran.r-project.org/web/packages/ggraph/index.html), and [ggplot](https://ggplot2.tidyverse.org/reference/ggplot.html).
+
+## Progress
+### Week 1
 We spent the first week learning about our research interest ([bees](https://www.wikiwand.com/en/Bee)) and exploring the [Global Biotic Interactions (GloBI)](https://www.globalbioticinteractions.org/about) dataset. Our data come from literature reports, human observations, and natural history specimen. 
-Goal of GloBI is to structure data into a usable form (csv., tsv.). \
-<img src="http://clipart-library.com/images/yTk4eXgTE.png" width="200" />\
-During our weekly meeting with the sponsors, we gained substantial knowledge about bees, especially the vital role it plays in global biotic interactions. In particular, we want to focus our research on the interactions between bees and plants. Interactions are directional, i.e. "Apis Mellifera pollinates brasicca rapa" will be categorized into source, interaction, and target, respectively.\
+Goal of GloBI is to structure data into a usable form (csv., tsv.). 
+
+![](https://github.com/angelchen7/ucsb-ds-capstone-2021.github.io/blob/main/ucsb_ds_capstone_projects_2021/projects/ccber/bee.PNG?raw=true)
```

Comment:
Same here
Markdown can embed images directly from the same directory: https://jupyterbook.org/content/figures.html
For the image  `some-image.png` in the same directory (you already have it), replace the HTML with `![some-text](`some-image.png)`